test/support/page_objects/common.js

```diff
@@ -351,11 +351,13 @@ export default class Common {
   async clickConfirmOnModal() {
     this.debug('Clicking modal confirm');
     await this.findTestSubject('confirmModalConfirmButton').click();
+    await PageObjects.common.sleep(1000);
```

Comment:
Just wanted to 2x check this was intentionally getting shipped, all the `sleep` and `tryForTime` confuse me.